hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/ConfigFactory.java

```diff
@@ -41,4 +50,26 @@ private String getProperty(Configs opt) {
         return propertyFactory.getContextualProperty(opt.getName(), opt.getDefaultValue()).getValue().toString();
     }
 
+    private DynamicPropertyFactory getDynamicPropertyFactory() {
+        if (!ConfigurationManager.isConfigurationInstalled()) {
+            PolledConfigurationSource source = new URLConfigurationSource();
+            AbstractPollingScheduler scheduler = createDisabledPollingScheduler();
+            ConfigurationManager.install(new DynamicConfiguration(source, scheduler));
+        }
+        return DynamicPropertyFactory.getInstance();
+    }
+
+    private static AbstractPollingScheduler createDisabledPollingScheduler() {
+        return new AbstractPollingScheduler() {
+            @Override
+            protected void schedule(Runnable pollingRunnable) {
+                LOGGER.info("Periodical polling of a configuration source is turned off!");
```

Comment:
Is this going to be logged once or every x period? if the second one then maybe we should change log level to debug.

Reply:
this is only called once when the configuration is created. I gave an entry on info level so that nobody would be surprised that it does not work and there was a trace of it